website/docs/guides/1.18.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,30 @@
+---
+page_title: "Upgrade Guide 1.18.0"
+---
+
+# MongoDB Atlas Provider 1.18.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.18.0 has a number of new and exciting features.
+
+**New Resources, Data Sources, and Features:**
+
+- Sharded and geo-sharded clusters defined with `mongodbatlas_advanced_cluster` are now capable of scaling the instance size and disk IOPS independently for each individual shard. For more details and migration guidelines please reference [advanced_cluster - Migration to new sharding schema and leveraging Independent Shard Scaling](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/advanced-cluster-new-sharding-schema). As part of these changes 2 new attributes have been included:
+    - New attribute `replication_specs.*.zone_id` is added in the resource and data sources to enable identifying the zone of each `replication_specs` object.
+    - New attribute `use_replication_spec_per_shard` defined in both data sources to configure if the users want to obtain `replication_specs` objects for each shard.
+
+**Deprecations:**
+
+- Deprecations in `mongodbatlas_advanced_cluster` resource and data sources:
+    - `replication_specs.*.num_shards`: The `replication_specs` list now supports defining an object for each inidividual shard and this new schema is favoured over using `num_shards` attribute. For more details and migration guidelines please reference [advanced_cluster - Migration to new sharding schema and leveraging Independent Shard Scaling](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/advanced-cluster-new-sharding-schema).
+    - `disk_size_gb`: The same attribute is now defined under `replication_specs.*.region_configs.*.(electable_specs|analytics_specs|read_only_specs).disk_size_gb`. Replacing the root value into existing inner specs will have no change in the underlying cluster. The motivation behind this change in location is to align with the new API schema and facilitate new features related to independent storage size scaling in the future.
```

Comment:
Does this mean that if a cluster is currently deployed with a value of "X" set to `disk_size_gb` and that same value is set to the new field location, the cluster configuration will not change?

Reply:
that is correct